automatic/intunewinapputil/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Win32 Content Prep Tool
+
+      Manage Windows Apps (.intunewin) with Intune
```

Comment:
Don't intend the lines in this file, you are making all of the lines become code blocks in this case.